tests/Autodesk.Forge.Core.Test/TestForgeConfiguration.cs

```diff
@@ -51,6 +51,23 @@ public void TestValuesFromLegacyEnvironment()
             Assert.Equal("blabla", config.Value.ClientSecret);
         }
 
+        [Fact]
+        public void TestValuesFromAPSEnvironment()
+        {
+            Environment.SetEnvironmentVariable("APS_CLIENT_ID", "bla");
+            Environment.SetEnvironmentVariable("APS_CLIENT_SECRET", "blabla");
+            var configuration = new ConfigurationBuilder()
+                .AddAPSAlternativeEnvironmentVariables()
```

Comment:
what if I call `AddForgeAlternativeEnvironmentVariables` too?

Reply:
We should mark the old stuff with `[Obsolete]` so you get a warning from the compiler.

Reply:
If you call that it works too, but I will mark it as [Obsolete].

Thanks.